tdrs-frontend/src/actions/createXLSReport.js

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios'
+import base64toFile from 'node-base64-to-file'
+import { useEventLogger } from '../utils/eventLogger'
+export const SET_PARSE_ERRORS = 'SET_PARSE_ERRORS'
+export const SET_PARSE_ERRORS_ERROR = 'SET_PARSE_ERRORS_ERROR'
+export const FETCH_PARSE_ERRORS = 'FETCH_PARSE_ERRORS'
+
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL
+
+/* 
+Get a list of parse errors for a given file id from the backend using the
+`/parsing/parse_errors/{id}` endpoint.
+*/
+export const getParseErrors = () => async (dispatch) => {
+  dispatch({
+    type: FETCH_PARSE_ERRORS,
+  })
+  try {
+    const response = await axios.get(
+      `${process.env.REACT_APP_BACKEND_URL}/parsing/parsing_errors/`,
+      {
+        responseType: 'json',
+      }
+    )
+    dispatch({
+      type: SET_PARSE_ERRORS,
+      payload: {
+        data: response?.data,
+      },
+    })
+    const data_json = response?.data
+    const blob = b64toBlob(data_json.xls_report, 'blob')
+    const blobUrl = URL.createObjectURL(blob)
+    const link = document.createElement('a')
+    link.href = blobUrl
+    link.download = 'results.xlsx'
+    document.body.append(link)
+    link.click()
+    link.remove()
+  } catch (error) {
+    dispatch({
+      type: SET_PARSE_ERRORS_ERROR,
```

Comment:
does there need to be a complementary reducer to handle these action dispatches?